erpnext/stock/report/stock_ageing/stock_ageing.py

```diff
@@ -32,6 +32,15 @@ def execute(filters=None):
 		if filters.get("show_warehouse_wise_stock"):
 			row.append(details.warehouse)
 
+		"""
+			add data for batch_no and package_tag if selected in filters
```

Comment:
this should be a single line comment, that starts with a #

Reply:
sure